api/src/routes/anime/animeRoutes.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import { animeController } from "@@root/controllers";
+import { extractPaginationParams, validation } from "@@middleware/index.js";
+
+const animeRoutes = express.Router();
+
+animeRoutes.route("/").get(extractPaginationParams, animeController.index);
+
+animeRoutes.route("/anime/id/:id")
+    .post(validation.id, animeController.create)
```

Comment:
You can move this to the root route where the `index` controller function is called since there is no ID used by an entity that hasn't been created